pkg/util/usage/usage.go

```diff
@@ -44,6 +44,10 @@ func Usage(printAll bool, configs ...interface{}) error {
 			ptr := v.Pointer()
 			field, ok = fields[ptr]
 			if ok {
+				if isFieldHidden(field) {
```

Comment:
I'm not sure this is the right place where to put this logic. We enter this `else if` branch if the field has no category override, but we should apply this logic anyway. Why not just moving this logic above `if override, ok := fieldcategory.GetOverride(fl.Name); ok {` so that it's unrelated from the field category logic?